cypress/support/config/settings.js

```diff
@@ -114,15 +114,27 @@ module.exports = () => ({
       photoGalleryPage: {
         environments: {
           live: {
-            paths: ['/afaanoromoo/oduu-50716382'],
-            enabled: false,
+            paths: [
+              '/pidgin/tori-49221071',
+              '/mundo/deportes-36935058', // PGL LTR
+              '/persian/magazine-49281981', // PGL RTL
```

Comment:
These URLs should be configured for the corresponding service e.g. pidgin, mundo and persian, rather than in afaanoromoo config.

Reply:
How could I miss this 

<img src="https://media.giphy.com/media/6yRVg0HWzgS88/giphy.gif" />